sound.js

```diff
@@ -26,7 +26,32 @@ function Sound(filename, basePath, onError, options) {
     }
   }
 
+  this.registerOnPlay = function() {
+    if (this.onPlaySubscription != null) {
+      console.warn('On Play change event listener is already registered');
+      return;
+    }
+
+    if (!IsWindows) {
+      this.onPlaySubscription = eventEmitter.addListener(
+        'onPlayChange',
+        (param) => {
+          const { isPlaying, key } = param;
+          if (key == this._key) {
+            if (isPlaying) {
+              this._playing = true;
+            }
+            else {
+              this._playing = false;
+            }
+          }
+        },
+      );
+    }
+  }
+
   this._loaded = false;
+  this._playing = false;
   this._key = nextKey++;
```

Comment:
I found that `nextKey` was declared as `var` and eventEmitter was declared as `const`. If you want to comply with the convention of this project. You might need to change the declaration part of `eventEmitter` to `var`.
(Personally, I think `var` should be removed in JS, especially RN environment. It's another story for web development). (fyi, @rjw0205)